src/backend/distributed/commands/multi_copy.c

```diff
@@ -21,11 +21,11 @@
  * table is append-partitioned, metadata changes are rolled back on the master
  * node, but shard placements are left on the workers.
  *
- * By default, COPY uses normal transactions on the workers. In the case of
+ * By default, citus uses normal transactions on the workers. In the case of
```

Comment:
Done that. I tried to emphasize that, COPY does not use normal transactions by default not because it wants to, but Citus' multi shard delete protocol has 1pc as default.